cmd/esc/cli/testdata/env-set-secret.yaml

```diff
@@ -16,6 +16,8 @@ run: |
   esc env get default/test
   esc env set default/test password true --secret
   esc env get default/test
+  echo 'mysecret' | esc env set default/test password --secret
```

Comment:
As mentioned in the PR description, I can't get this to work. It's as if `echo 'mysecret'` evaluates to an empty string because that's what the value of `esc env set ...` ends up getting. However, building a local version of the CLI and executing the same thing works as expected, so I am not sure what is going on with the test runner.